core/crypto/crypto.h

```diff
@@ -65,6 +65,10 @@ class X509Certificate : public Resource {
 	virtual Error load(String p_path) = 0;
 	virtual Error load_from_memory(const uint8_t *p_buffer, int p_len) = 0;
 	virtual Error save(String p_path) = 0;
+	virtual Error load_from_string(String p_string_cert) = 0;
+	virtual String save_to_string() const = 0;
+	virtual String _get_cert() const = 0;
```

Comment:
This has the exact same signature of `save_to_string()`. I think it should be removed and `save_to_string` used as the getter.
Not sure if the same thing can be done with `_set_cert` and `load_from_string` (which returns a value, so I'm not sure the ClassDB is okay with it but should be tested)

Reply:
I think Godot should function with removing _set_cert  and load_from_string.